src/components/feature.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+const Feature = (props) => (
```

Comment:
Use can use destruction in it like
```
const Feature = ({ feature: { name } }) => (
```
and directly use `name` instead of `props.feature.name`